### PR TITLE
[metricsadvisor] drop support for 3.6 (Azure#25168)

### PR DESCRIPTION
**Pull request by wonder6845**
# Description

Please add an informative description that covers that changes made by the pull request and link all relevant issues.

If an SDK is being regenerated based on a new swagger spec, a link to the pull request containing these swagger spec changes has been included above.

# All SDK Contribution checklist:
- [x] **The pull request does not introduce [breaking changes]**
- [x] **CHANGELOG is updated for new features, bug fixes or other significant changes.**
- [x] **I have read the [contribution guidelines](https://github.com/Azure/azure-sdk-for-python/blob/main/CONTRIBUTING.md).**

## General Guidelines and Best Practices
- [x] Title of the pull request is clear and informative.
- [x] There are a small number of commits, each of which have an informative message. This means that previously merged commits do not appear in the history of the PR. For more information on cleaning up the commits in your PR, [see this page](https://github.com/Azure/azure-powershell/blob/master/documentation/development-docs/cleaning-up-commits.md).

### [Testing Guidelines](https://github.com/Azure/azure-sdk-for-python/blob/main/CONTRIBUTING.md##building-and-testing)
- [x] Pull request includes test coverage for the included changes.

### CODE REVIEW

File: sdk/metricsadvisor/azure-ai-metricsadvisor/CHANGELOG.md [hunk 1]
```diff
@@ -10,6 +10,7 @@
 
 ### Other Changes
 - Python 2.7 is no longer supported. Please use Python version 3.6 or later.
```
Comment (wonhyeongseo) [inline sdk/metricsadvisor/azure-ai-metricsadvisor/CHANGELOG.md:12]: > Python 2.7 is no longer supported.
We may remove the second sentence as it is reflected in the next list item. Great job, my friend.

Comment (kristapratico) [reply sdk/metricsadvisor/azure-ai-metricsadvisor/CHANGELOG.md:12]: Ah looks like we haven't released this package in a bit. Personally I'd combine the bullet points, but call out the versions we dropped like. "Python 2.7 and 3.6 no longer supported...".

File: sdk/metricsadvisor/azure-ai-metricsadvisor/README.md [hunk 2]
```diff
@@ -10,6 +10,8 @@ Metrics Advisor is a scalable real-time time series monitoring, alerting, and ro
 
 ## _Disclaimer_
 
+_Python 3.7 or later is required to use this package. For more details, please refer to [Azure SDK for Python version support policy](https://github.com/Azure/azure-sdk-for-python/wiki/Azure-SDKs-Python-version-support-policy)._
```
Comment (kristapratico) [inline sdk/metricsadvisor/azure-ai-metricsadvisor/README.md:13]: Like with other packages, I think we can omit his from the README. Dropping 2.7 was a special case.

**Review by wonder6845 — COMMENTED** [review]
Thanks for your feedback. I'll take that into consideration soon

File: sdk/metricsadvisor/azure-ai-metricsadvisor/CHANGELOG.md (hunk 1 above)
Comment (wonder6845) [reply sdk/metricsadvisor/azure-ai-metricsadvisor/CHANGELOG.md]: Thanks for your feedback. I'll take that into consideration soon.

File: sdk/metricsadvisor/azure-ai-metricsadvisor/CHANGELOG.md [hunk 3]
```diff
@@ -9,7 +9,7 @@
 ### Bugs Fixed
 
 ### Other Changes
-- Python 2.7 is no longer supported. Please use Python version 3.6 or later.
+- This version and all future versions will require Python 3.7+. Python 2.7 and 3.6 no longer supported.
```
Comment (kristapratico) [inline sdk/metricsadvisor/azure-ai-metricsadvisor/CHANGELOG.md:12]: typo

```suggestion
- This version and all future versions will require Python 3.7+. Python 2.7 and 3.6 are no longer supported.
```

File: sdk/metricsadvisor/azure-ai-metricsadvisor/README.md [hunk 4]
```diff
@@ -10,7 +10,8 @@ Metrics Advisor is a scalable real-time time series monitoring, alerting, and ro
 
 ## _Disclaimer_
 
-_Azure SDK Python packages support for Python 2.7 has ended 01 January 2022. For more information and questions, please refer to https://github.com/Azure/azure-sdk-for-python/issues/20691_
```
Comment (kristapratico) [inline sdk/metricsadvisor/azure-ai-metricsadvisor/README.md:13]: Sorry if I was unclear here - let's keep the line about 2.7 and remove the line about 3.7. :)

**Review by kristapratico — COMMENTED** [review]
Few small things - thanks!

**Review by kristapratico — APPROVED** [review]
Looks great! Thanks for your contribution 😸

**Pull request merged**